index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
Please use descriptive meta tags in the head section. Your code has none. Please make sure you add those.